src/read_write/ply.rs

```diff
@@ -454,7 +454,10 @@ impl NodeWriter<PointsBatch> for PlyNodeWriter {
                         (
                             &k[..],
                             match data {
+                                AttributeData::I64(_) => panic!("Can't write i64 attribute to PLY"),
```

Comment:
Shall we extend `DataType` with `Int64` and `Uint64` and its corresponding `longlong/int64` and `ulonglong/uint64` `from_str` methods instead? I know its not officially supported, but we could still support it to make debugging easier...

Reply:
Okay

Reply:
Done